src/openvic-simulation/pop/Pop.hpp

```diff
@@ -83,6 +83,22 @@ namespace OpenVic {
 		fixed_point_t PROPERTY(life_needs_fulfilled);
 		fixed_point_t PROPERTY(everyday_needs_fulfilled);
 		fixed_point_t PROPERTY(luxury_needs_fulfilled);
+		fixed_point_t PROPERTY(artisanal_income);
+		fixed_point_t PROPERTY(factory_worker_income);
+		fixed_point_t PROPERTY(factory_owner_income);
+		fixed_point_t PROPERTY(unemployment_subsidies_income);
+		fixed_point_t PROPERTY(pensions_income);
+		fixed_point_t PROPERTY(government_salary_administration_income);
+		fixed_point_t PROPERTY(government_salary_education_income);
+		fixed_point_t PROPERTY(government_salary_military_income);
+		fixed_point_t PROPERTY(event_and_decision_income);
+		fixed_point_t PROPERTY(loan_interest_payments);
+
```

Comment:
No need for new lines here.

Reply:
Following the style of the repo, there should never be more than a single blank line regardless of the context